src/lib/timeseries.spec.js

```diff
@@ -330,7 +336,7 @@ describe('Aggregation', () => {
   describe('computeCO2Timeseries', () => {
     it('should return the total CO2 for all timeseries', () => {
       expect(computeCO2Timeseries(aggregatedTimeseries)).toBe(
-        135.91453799999996
+        138.75402599999998
```

Comment:
why this change?

Reply:
Cela est dû à cet ajout : https://github.com/cozy/coachCO2/pull/80/files/ff37f5d66b2158e5ca0763a3f64a1a743ef150a7#diff-87d0acddc8e46544e133dec4844ab48c19a3d1a715caeedfc6c12cfc28e4f373R153